ops_boba/monitor/services/utilities/optimismEnv.js

```diff
@@ -81,6 +79,8 @@ const OVM_L2_CROSS_DOMAIN_MESSENGER =
 
 const L1_BLOCK_CONFIRMATION = env.L1_BLOCK_CONFIRMATION || 0
 
+const NUMBER_OF_BLOCK_TO_FETCH = env.NUMBER_OF_BLOCK_TO_FETCH || 10000000
```

Comment:
```suggestion
const NUMBER_OF_BLOCKS_TO_FETCH = env.NUMBER_OF_BLOCKS_TO_FETCH || 10000000
```

this env var should also be documented in ops_boba/monitor/README.md